chia/cmds/show.py

```diff
@@ -1,13 +1,17 @@
-from typing import Any, Optional, Union, Dict
+import time
+from typing import Any, Callable, Dict, List, Optional, Union
 
-from chia.types.blockchain_format.sized_bytes import bytes32
 import click
 
-from chia.util.network import is_trusted_inner
+from chia.consensus.block_record import BlockRecord
```

Comment:
we are importing extra stuff here. Can we import it inside the functions instead for efficiency?

Reply:
Yep